### ISSUE TITLE
Program returns opposite of 'best model' [BUG]

### ISSUE THREAD

**Issue by jjpass00** [franneck94/TensorCross#43]
In `_run_search()` -func in `search_cv.py` as you can see the code determines that the best model is the one which has the largest mean of scores:

```
mean_val_scores = np.mean(self.results_["val_scores"], axis=0)
best_run_idx = np.argmax(mean_val_scores)
```
Essentially this returns wrong model, if somekind of _loss_ is used in the calculation of `val_scores` (because with loss smaller is bettter). This wouldn't be a problem (it would be a feature) if it wasn't stated in the documentation that
> The grid search is evaluated by:
> - The validation loss value, if no metrics are passed to model.compile()
> - The validation score of the last defined metric in model.compile()

This kind of implies that you can use either _score_ or _loss func_ and you still get the best model which isn't the case.

Best way to fix this would be to add a '`mode`' argument to the function/method. This parameter would determine which is desirable; maximizing or minimizing the *score*.

**Comment by franneck94** [comment]
Should be fixed now
